lib/tester/AlexaTest.ts

```diff
@@ -218,9 +218,9 @@ export class AlexaTest {
             lambdaHandler: 'handler',
             timeoutMs: 5000,
         }).then((response) => {
-            if (response.toJSON) {
-                response = response.toJSON();
-            }
+            // if (response.toJSON) {
```

Comment:
Can you elaborate?

Reply:
I was getting a compile error `Property 'toJSON' does not exist on type 'unknown'.` so disabled this and didn't get an errors running tests.  Does this compile for you?

Reply:
I will test this today and then merge the PR